src/object/srv_obj_migrate.c

```diff
@@ -35,13 +35,11 @@
  * the moment, will adjust it later if needed.
  */
 #define MIGRATE_DEFAULT_MAX_SIZE	(1 << 28)
```

Comment:
is the intent of this patch to change the default maximum size in bytes to 256MiB as mentioned in the documentation files? It is still 512MiB here.

Reply:
Nope, i actually did not intend to change any default in this PR (yet).
2^28 bytes = 268435456 bytes = 256MiB
Am i missing something?

Reply:
oops yeah, my mistake on 2^28. Oof.